Update questions.json

DSeanHD opened the pull request:
Added a JavaScript question titled: "What would be the result of 3+2+”7″?"

DSeanHD committed:
Update questions.json

Added a JavaScript question titled: "What would be the result of 3+2+”7″?"

lloyd-december reviewed (changes requested):
This is great, 
but have this part "3+2+”7″? " be as a code snippet and have it console.log() or something, so its an actual javascript code

And the text  can be:
What is the result of the code below

DSeanHD committed:
Update questions.json

Added a codeSnippet to the question and made the text: "What would be the result of the code below"

DSeanHD commented:
Got it! Made the changes just now

llo7d commented:
@DSeanHD Added. Good stuff
